Source/WebCore/editing/Editor.cpp

```diff
@@ -2279,6 +2282,9 @@ void Editor::setComposition(const String& text, const Vector<CompositionUnderlin
         }
     }
 
+    if (auto *cache = m_document.existingAXObjectCache())
```

Comment:
WebKit C++ coding style requires `auto* foo` instead of `auto *foo`.

Reply:
Applied to both instances of this. Surprised the linter missed that, it's a habit of my other codebases. Thanks for finding it though.